recipes/gempy_engine/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gempy_engine" %}
```

Comment:
Let's make this `gempy-engine` to match how it's installed via `pip`.

Reply:
Corrected